module/zfs/zvol.c

```diff
@@ -319,6 +320,11 @@ zvol_create_cb(objset_t *os, void *arg, cred_t *cr, dmu_tx_t *tx)
 
 	error = zap_update(os, ZVOL_ZAP_OBJ, "size", 8, 1, &volsize, tx);
 	ASSERT(error == 0);
+
+#if !defined(_KERNEL)
+	VERIFY(uzfs_zvol_create_meta(os, volblocksize, volblocksize,
+	    volblocksize, tx) == 0);
```

Comment:
lets leave metavolblocksize as 512, the lowest at which iscsiblocklength can be configured